substratools/utils.py

```diff
@@ -101,9 +101,17 @@ def load_interface_from_module(module_name, interface_class, interface_signature
         )
 
     # find interface class
+    found_interfaces = []
     for _, obj in inspect.getmembers(module, inspect.isclass):
-        if issubclass(obj, interface_class):
-            return obj()  # return interface instance
+        if issubclass(obj, interface_class) and obj != interface_class:
+            found_interfaces.append(obj)
+
+    if len(found_interfaces) == 1:
+        return found_interfaces[0]()  # return interface instance
+    elif len(found_interfaces) > 1:
+        raise exceptions.InvalidInterfaceError(
+            f"Multiple interfaces found in module '{module_name}': {found_interfaces}"
```

Comment:
Change the error message no one knows what interface mean

Reply:
Multiple opener sub-classes defined in the opener.py file, this is not allowed.

Reply:
The opener and the different subclasses will appear in `module_name` and `found_interfaces`

Reply:
I disagree you don't have the info of interface_class which is in this case opener

Reply:
Please add at least interface_class to the debug print statement